tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -757,6 +995,87 @@ def test_positive_provision_rhev_with_custom_compute_settings(self):
 
         :Caseautomation: notautomated
```

Comment:
Shouldn't this be changed? @san7ket @ldjebran from `notautomated` to `:CaseAutomation: Automated` (or sumply removed to use the default?)